resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
Just capture and store context around the event? If you have `<div data-trackable="foo">foo</div>` and that sort of thing then you can understand what's going on in a lot of detail.